test/system/Change_Docker_Config.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+
```

Comment:
I think you want a `set -e` here

Reply:
using ` sed -e` , when needed.

Reply:
done, for both ` Change_Docker_Config_Master.sh`, `Change_Docker_Config_Slave.sh`.